packages/core/phase2/lib/generateOperations/generateOperations.test.ts

```diff
@@ -341,6 +284,31 @@ describe("generateOperations", () => {
     expect(exceptions).toHaveLength(0)
   })
 
+  it("Should return no operations when there are no recordable offences", () => {
+    mockedAreAllResultsOnPnc.mockReturnValue(false)
+    const aho = {
+      Exceptions: [],
+      AnnotatedHearingOutcome: {
+        HearingOutcome: {
+          Case: {
+            HearingDefendant: {
+              Offence: [
+                {
```

Comment:
This offence is recordable and the test is not covering getOperations.ts line 76. You can check it by adding a breakpoint on line 76 and running your test in debug mode.

To make this offence non-recordable, you can set `OffenceCategory` to `B7`. (see [isRecordableOffence.ts](https://github.com/ministryofjustice/bichard7-next-core/blob/17d9be116f6c45f24d5ae29818e1a9d16c67ac96/packages/core/phase2/lib/isRecordableOffence.ts#L7))